changelogs/unreleased/gh-8825-luajit-fixes.md

```diff
@@ -7,3 +7,10 @@ were fixed as part of this activity:
 * Fixed recording of `BC_VARG` with unused vararg values.
 * Initialization instructions on trace are now emitted only for the first
   member of a union.
+* Prevent integer overflow while parsing long strings.
+* Fixed various `^` operator inconsistencies.
```

Comment:
This also includes math.pow() function. However, feel free to ignore.

Reply:
Fixed. BTW, I took the entry from #8985.

Reply:
I rethought and decided that this description is better.